frontend/public/components/dashboard/project-dashboard/details-card.tsx

```diff
@@ -26,9 +30,19 @@ export const DetailsCard: React.FC = () => {
           <DetailItem isLoading={!obj} title="Name">
             {getName(obj)}
           </DetailItem>
+          <DetailItem isLoading={!obj} title="Requester">
+            {getRequester(obj) || <span className="text-muted">No requester</span>}
+          </DetailItem>
           <DetailItem isLoading={!obj} title="Labels">
             <div className="co-project-dashboard__details-labels">
-              {obj && <LabelList kind={ProjectModel.kind} labels={obj.metadata.labels} />}
+              {obj && <LabelList kind={ProjectModel.kind} labels={firstThreelabels} />}
```

Comment:
`obj` can't be null because it will blow up above.

```suggestion
              <LabelList kind={ProjectModel.kind} labels={firstThreelabels} />
```